Region settings

nop33 opened the pull request.

mvaivre reviewed (commented):
Great work, thanks. A few comments:

- The number formatting isn't applied to token amounts:
<img width="124" alt="image" src="https://github.com/user-attachments/assets/41c4e774-d1b8-4b47-b708-9e9257b75676" />


- A search field would be great for such long lists. WDYT? Let's create an issue? 

- Not directly liked to this PR but it highlighted the issue : let's align text to left here :)
<img width="227" alt="image" src="https://github.com/user-attachments/assets/eaa8f49e-ef87-476f-949d-aaa75d822bd5" />

nop33 commented:
> The number formatting isn't applied to token amounts:

Fixed ✅ 

> A search field would be great for such long lists. WDYT? Let's create an issue?

- https://github.com/alephium/alephium-frontend/issues/1215

> Not directly liked to this PR but it highlighted the issue : let's align text to left here :)

Fixed ✅

mvaivre reviewed (approved):
Should be good now based on our discussion!
